types/context.go

```diff
@@ -220,6 +225,10 @@ func (c Context) WithMinimumFees(minFees Coins) Context {
 	return c.withValue(contextKeyMinimumFees, minFees)
 }
 
+func (c Context) WithConsensusParams(params *abci.ConsensusParams) Context {
```

Comment:
I think this was recently removed from the context because we have it in the BaseApp too -- which I think makes this somewhat confusing, but I guess we cannot get around this.